tests/http_mock.py

```diff
@@ -12,25 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Copy of googleapiclient.http's mock functionality."""
+"""HTTP helpers mock functionality."""
 
-import httplib2
 
-# TODO(craigcitro): Find a cleaner way to share this code with googleapiclient.
+class ResponseMock(dict):
+    """Mock HTTP response"""
```

Comment:
In light of #128, the goal is to be **less** precise, at least until we define the common interface.